crud/common/sharding/init.lua

```diff
@@ -130,6 +130,17 @@ function sharding.result_needs_sharding_reload(err)
     return err.class_name == sharding_utils.ShardingHashMismatchError.name
 end
 
+function sharding.batching_result_needs_sharding_reload(errs, tuples_count)
+    local sharding_errs_count = 0
+    for _, err in ipairs(errs) do
+        if err.class_name == sharding_utils.ShardingHashMismatchError.name then
+            sharding_errs_count = sharding_errs_count + utils.table_count(err.operation_data)
```

Comment:
Is it total amount of fields, not tuples? Why we compare it with amount of tuples?

Reply:
This is my mistake, I forgot to change multiple tuples to one tuple/object in the `operation_data` field for `ShardingHashMismatchError` error. Now it is fixed.